specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -359,24 +387,28 @@ namespace Microsoft.Windows.PushNotifications
     };
 
     [flags]
-    enum PushNotificationRegistrationKind
+    enum PushNotificationRegistrationOptions
```

Comment:
No need to rename this. Lets keep as is

Reply:
In the PushNotifications PR, Raymond asked us to change this to *Options so I was just updated the spec with what is in main.

Reply:
Ah got it. Thanks for the context